lib/hexpm/web/controllers/dashboard_controller.ex

```diff
@@ -28,6 +28,46 @@ defmodule Hexpm.Web.DashboardController do
     end
   end
 
+  def keys(conn, _params) do
+    user = conn.assigns.current_user
+    keys = Keys.all(user)
+
+    render_keys(conn, keys)
+  end
+
+  def revoke_key(conn, %{"name" => name} = params) do
+    user = conn.assigns.current_user
+
+    case Keys.remove(user, params["name"], audit: audit_data(conn)) do
```

Comment:
since you're pattern matching on `#{"name" => name}` we can use `name` binding here. This will get rid of the warning. Please make sure there are no other warnings by e.g. running `mix compile --force --warnings-as-errors` (I'm gonna add that check to CI later)